client/cypress/integration/visualizations/sankey_sunburst_spec.js

```diff
@@ -35,6 +37,12 @@ describe('Sankey and Sunburst', () => {
     cy.getByTestId('VisualizationPreview').find('svg').should('exist');
     cy.getByTestId('EditVisualizationDialog').contains('button', 'Save').click();
     cy.getByTestId('QueryPageVisualizationTabs').contains('li', visualizationName).should('exist');
+
+    cy.getByTestId('QueryPageShowDataOnly').click();
+
+    // wait a bit before taking snapshot
+    cy.wait(1000); // eslint-disable-line cypress/no-unnecessary-waiting
+    cy.percySnapshot('Visualizations - Sunburst', { widths: [viewportWidth] });
```

Comment:
Perhaps you can skip the wait by instead of clicking "Show Data Only", you can `cy.visit('queries/${id}')` and then snap?

Reply:
I added this timeout to wait for chart to load a query results data. "Show Data Only" is actually a link, so it should be completely equal to `cy.visit(...)`. But I may give it a try

Reply:
Seems there is no difference - `cy.visit` need `cy.wait` as well. I prefer to keep current variant because 1) it already works 2) it's a bit simpler :slightly_smiling_face: